src/components/content/content.spec.tsx

```diff
@@ -2,22 +2,24 @@ import React from "react";
 import { ThemeProvider } from "styled-components";
 import { mount, shallow } from "enzyme";
 import mintTheme from "../../style/themes/mint";
-import Content from "./content.component.js";
+import Content from ".";
```

Comment:
```suggestion
import Content, { ContentProps } from ".";
```

instead of import from line 17